app/client/src/features/form/components/sections/Profile.js

```diff
@@ -34,8 +68,48 @@ function Profile() {
         label="Link"
         placeholder="mycoolportfolio.com/myname"
       />
+        <Label>Summarys</Label>
+        {summarys.map((summary, i) => (
+            <Row key={i}>
+                <MiniInput
+                    type="text"
+                    name={`basics.summarys[${i}]`}
+                    placeholder="Experienced Regional Manager with a demonstrated history of working in the paper supply industry."
+                    component="input"
+                />
+                <ButtonRow hidden={i !== summarys.length - 1}>
+                    <RoundButton
+                        inverted
+                        disabled={i !== summarys.length - 1}
+                        type="button"
+                        onClick={() => addProfileSummary()}
+                    >
+                        <Icon type="add" />
+                    </RoundButton>
+                    <RoundButton
+                        inverted
+                        disabled={summarys.length === 1}
+                        type="button"
+                        onClick={() => removeProfileSummary()}
+                    >
+                        <Icon type="remove" />
+                    </RoundButton>
+                </ButtonRow>
+            </Row>
+        ))}
```

Comment:
My main concern right now is having this summaries field be present in the form for the other templates will be confusing since it won't work for any template besides 10. I don't think we need to wait on adding support for this summary section to all templates, but for now I think we can conditionally render this section only if the current template is `10`. 

Let me know if you have any other thoughts on that.

Reply:
Yeah, that makes more sense. Thanks.

Reply:
Done